charts/kuberpult/values.yaml

```
@@ -61,6 +61,9 @@ argocd:
 datadogTracing:
   enabled: false
   debugging: false
-  metrics: false
+
+dogstatsdMetrics:
+  enabled: false
+  address: "127.0.0.1:8125"
```

I would add a link to the documentation here, to explain why port 8125. Explaining that here in text would be nice for someone like me, but I would consider it not critical.

```suggestion
  # dogstatsD listens on port udp:8125 by default.
  # https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent#agent
  address: "127.0.0.1:8125"
```

---

Fair enough, I will add this